src/software/jetson_nano/display/robot_ui.py

```diff
@@ -156,20 +171,9 @@ def init_redis():
     def start_polling(robot_ui):
         robot_ui.poll_redis()
 
-    # start redis server
-    cmd = "cd ../linux_configs/redis_config && sudo docker-compose up -d"
-    st, out = subprocess.getstatusoutput(cmd)
-    init_redis()
-
-    robot_ui = RobotUi()
+    robot_ui = RobotUi(boot_screen_path=args["path_to_boot_screen"])
     thread = Thread(target=start_polling, args=(robot_ui,))
     thread.start()
 
-    print("Press any key to exit...")
-    input()
-    robot_ui.stop()
-    thread.join()
-
-    # stop redis server
-    cmd = "cd ../linux_configs/redis_config && sudo docker-compose down"
-    st, out = subprocess.getstatusoutput(cmd)
+    while True:
+        time.sleep(1)
```

Comment:
what is this for?

Reply:
This main thread needs to hang forever so that the robot UI thread will run in the background